tests/test_enterprise.yaml

```diff
@@ -35,3 +35,37 @@
     - url: '/enterprise/v1/catalogs'
     - method: 'GET'
     - expected_status: [400]
+
+- test:
+    - name: 'Enterprise catalog endpoint returns HTTP 200'
+    - url: '/enterprise/v1/enterprise-catalogs'
+    - method: 'GET'
+    - headers: {'Authorization': 'aeiou'}
+    - expected_status: [200]
+
+- test:
+    - name: 'Enterprise catalog by UUID endpoint returns HTTP 200'
+    - url: '/enterprise/v1/enterprise-catalogs/0123456789abcdefg'
+    - method: 'GET'
+    - headers: {'Authorization': 'aeiou'}
+    - expected_status: [200]
+
+- test:
+    - name: 'Program by UUID endpoint returns HTTP 200'
+    - url: '/enterprise/v1/programs/0123456789abcdefg'
+    - method: 'GET'
+    - headers: {'Authorization': 'aeiou'}
+    - expected_status: [200]
+
+- test:
+    - name: 'Enterprise catalog POST is disabled'
+    - url: '/enterprise/v1/enterprise-catalogs'
+    - method: 'POST'
+    - headers: {'Authorization': 'aeiou'}
+    - expected_status: [405]
+
+- test:
+    - name: 'Enterprise catalog GET rejected without authorization'
+    - url: '/enterprise/v1/enterprise-catalogs'
+    - method: 'GET'
+    - expected_status: [400]
```

Comment:
I think the tests added here cover the code in edx-enterprise since each endpoint is serviced by the same view class. I'll add tests for the programs endpoint since we will be refactoring that endpoint into a separate view with your refactoring this sprint.